botan-src/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "botan-src"
+version = "0.1.0+2.15.0"
+authors = ["Rodolphe Breard <rodolphe@what.tf>"]
```

Comment:
This should probably be adjusted

Reply:
What kind of adjustments do you have in mind exactly?
The `[package]` section is required by Cargo.
The `name` follows a common pattern (see `openssl`, `openssl-sys`, `openssl-src`).
The `version` rational is explained in the PR description (0.1.0 for this new `botan-src` crate and 2.15.0 in the metadata since it is the Botan's version that is included).
As for the `authors`, I fully wrote this new `botan-src` crate.

Reply:
> I fully wrote this new botan-src crate

It's mostly @randombit 's source code that's inside the crate, no?

Reply:
Nope, the `botan-src` doesn't even exists yet, all its source code is in this PR.

Reply:
The crate itself is new but most of the code is not and was not written by you, as it includes the full source code of the botan C library.

Reply:
The `botan-src` is new and all of its code has been written be me. Please show me any line that hasn't.
As for Botan source code, it has nothing to do with it. This `botan-src` crate is a different tool that builds a specific version of the Botan source code and therefore does not claim any right on Botan itself.

Reply:
In some ways it is confusing especially as I maintain both the C++ library and the higher level crates but @breard-r is correct to list themselves as the author of this (`botan-src`) crate.